compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.h

```diff
@@ -25,15 +25,19 @@ namespace mlir::iree_compiler {
 /// attribute. These are used to override default pass heuristics at the
 /// function granularity.
 namespace LLVMGPUAttrNames {
-inline constexpr StringLiteral kNoReorderWorkgroups = "no_reorder_workgroups";
+inline constexpr StringLiteral kReorderWorkgroups = "reorder_workgroups";
 inline constexpr StringLiteral kNoReduceSharedMemoryBankConflicts =
     "no_reduce_shared_memory_bank_conflicts";
 } //  namespace LLVMGPUAttrNames
 
 struct LLVMGPUPipelineOptions {
+  enum reorderWorkGroupOption { None, Transpose, Swizzle };
```

Comment:
```suggestion
  enum ReorderWorkgroupsOption { None, Transpose, Swizzle };
```

Reply:
We should be careful so that we can distinguish the case when this disables workgroup reordering (when enabled globally) and when it's not set. I think we can use `std::optional<ReorderWorkgroupsStrategy>`